bitfinex-client/src/client/mod.rs

```diff
@@ -194,6 +241,41 @@ impl BitfinexClient {
         Ok(positions)
     }
 
+    #[instrument(name = "bitfinex_client.claim_position", skip(self), err)]
+    pub async fn claim_position(&self) -> Result<ClaimDetails, BitfinexClientError> {
+        let positions = self.get_positions().await?;
+
+        // get the BTC_USD_SWAP position
+        let mut symbol = Instrument::BtcUsdSwap.to_string();
+        if self.config.simulated {
+            symbol = "tTESTBTC:TESTUSD".to_string();
+        }
+        let btc_usd_position = positions
+            .iter()
+            .find(|p| p.symbol == symbol && p.status == "ACTIVE")
+            .ok_or(BitfinexClientError::NoActivePositionAvailable)?;
```

Comment:
a bit confused as to what should we return if there is no active position on the exchange? 
@sebastienverreault @bodymindarts